pages/once-upon-a-blog--9849zg/once-upon-a-blog--9849zg.md

```diff
@@ -0,0 +1,247 @@
+There are many other [Eleventy plugins](https://www.npmjs.com/search?q=eleventy-plugin) out there I wanted to try.
+I even started experimenting with an [asciidoctor](https://asciidoctor.org/) integration.
+However, if I want to beat procrastination, I really need to focus on the content and postpone this expedition in plugin land to another day. 
+
+### ![](./favicon-parcel.ico) Optimized with Parcel (v2)
+
+As explained in [this issue](https://github.com/11ty/eleventy/issues/272), Eleventy does not have an official asset pipeline.
+I don't think it's a problem, especially in my context where I don't use any CSS tool (Sass, Less, Tailwind and others).
+
+I strongly believe in decoupling things.
+Each layer should do one thing and do it well.
+Each layer should know as little as possible about other layers.
+In the context of my own site, I wanted to decouple these two layers: build content and optimize for performance.
+
+When the Eleventy is done building my content, I have a valid static site I could host as is.
+It's just not optimized for performance yet.
+I almost decided to go live without any optimizations, but it was too much fun to finally try [Parcel (v2)](https://v2.parceljs.org/).
+
+I was able to use the output directory from Eleventy as an input and generate an optimized site with:
+
+* Minified HTML
+* Minified CSS
+* Optimized images
+* Renamed assets with hashes so they can have "never expire" cache settings
+
+> You used a JavaScript bundler for a site that doesn't have any JS?
+
+Yes I did my friend.
+Parcel is more than just a JavaScript bundler.
+Unlike [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/guide/en/), it is really Web centric and it can start from an HTML entrypoint.
+I also chose it over lower level solution like [Gulp](https://gulpjs.com/) because it knows how HTML, CSS, JS and image files depend on each other.
+
+Parcel v2 is still in beta.
+I had to create a few custom plugins and I'm looking forward to the release of official plugins to add [other improvements on images](https://github.com/parcel-bundler/parcel/issues/3738).
+You can find more details about how I used it in this [pull request](https://github.com/hsablonniere/hsablonniere.com/pull/8).
+
+### ![](./favicon-clever-cloud.png) Hosted on Clever Cloud
+
+When I bought this domain in 2015, I hosted this site on [OVH](https://www.ovh.com/) with docker on a virtual private server I had.
+Configuring this by myself was fun at the beginning but it clearly was a pain to maintain and secure.
+
+A few years later, I moved it to [Netlify](https://www.netlify.com/).
+When it comes to deploying a static site, they really have a nice product, even when you only have a homepage and no articles.
+
+When I started working again on this in september, I decided to move the site to [Clever Cloud](https://www.clever-cloud.com/en/).
+First and obvious reason: I work for them.
```

Comment:
Saying "for them" and then using the first-person feels a bit weird.

Although you are working remotely, I would say this:
```suggestion
First and obvious reason: I work there.
```